### DISCUSSION TITLE
Why I cant create task list?

### DISCUSSION THREAD

**Discussion by aecceyhan** [community/community#69168] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello all,

I am trying to apply this documentation https://docs.github.com/en/issues/managing-your-tasks-with-tasklists/quickstart-for-tasklists

When I apply the documentation the resault is;

<img width="847" alt="image" src="https://github.com/community/community/assets/17556123/3793e24b-6b8d-40c0-9a0a-37a83bf6e069">

<img width="555" alt="image" src="https://github.com/community/community/assets/17556123/57e9414f-94ed-4836-92e3-382f2e3871cc">

However in the documentation;

<img width="1023" alt="image" src="https://github.com/community/community/assets/17556123/12370de4-baf9-4837-bc4f-c2e2d258a332">

So what I missing?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by gtjamesa** [comment, marked as answer]
Two things:

1. Taskslists as shown in your screenshot are in private beta, please see the note at the top of the page: https://docs.github.com/en/issues/managing-your-tasks-with-tasklists/about-tasklists
2. You can still use the previous tasklists, but your formatting is wrong. There should be a line break between the heading and the task

```markdown
### Tasks

- [ ] #243
- [ ] #242
- [ ] Draft task
````

**Comment by lklanswer** [comment]
See ((https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists).)
> We have currently paused onboarding to the beta. Tasklists build upon the previous iteration of [task lists]

**Reply by hansarh** [reply]
When can we expect this onboarding to open up again?

**Reply by Kurogoma939** [reply]
I would like to know that information too.
